src/platform/utilities/api/index.js

```diff
@@ -64,6 +64,7 @@ export function apiRequest(resource, optionalSettings = {}, success, error) {
           // If the user receives a 401 when trying to log out, it means their session
           // has expired.  Redirect them home.  In all other cases, redirect to login
           if (isLogout) {
+            window.localStorage.removeItem('hasSession');
```

Comment:
:+1:

Reply:
Maybe better to call `teardownProfileSession` to cover more bases.

Reply:
I think this is already covered here, but certainly doesn't hurt to do that here as well: https://github.com/department-of-veterans-affairs/vets-website/blob/2423155b25a9aded1377ef1df42ad0edb9ed7c34/src/platform/user/profile/actions/index.js#L57-L59